zipline/finance/performance/tracker.py

```diff
@@ -92,9 +93,12 @@ def __init__(self, sim_params, env=None):
         self.period_start = self.sim_params.period_start
         self.period_end = self.sim_params.period_end
         self.last_close = self.sim_params.last_close
-        first_day = self.sim_params.first_open
+        first_open = self.sim_params.first_open
+        self.day = first_open.tz_convert(trading.environment.exchange_tz)
```

Comment:
The assignment of `self.day` twice makes the `__init__` a little harder to read, at least semantically.
I'd suggest changing the above so that they are something like.
```
first_open = self.sim_params.first_open.tz_convert(trading.enviroment.exchange_tz)
```

and then use first_open instead of `self.day` in setting the 'real' self.day.